pybullet-helpers/src/pybullet_helpers/utils.py

```diff
@@ -49,15 +51,54 @@ def create_pybullet_block(
         physicsClientId=physics_client_id,
     )
 
-    # Create the body.
-    block_id = p.createMultiBody(
-        baseMass=mass,
-        baseCollisionShapeIndex=collision_id,
-        baseVisualShapeIndex=visual_id,
-        basePosition=position,
-        baseOrientation=orientation,
-        physicsClientId=physics_client_id,
-    )
+    if has_peg:
+        # Create a peg on top of the block for stacking tasks.
+        peg_height = 0.05
```

Comment:
usually I am a big proponent of avoiding redundant code, but in this case, I think it is actually better to create a new `create_pybullet_block_with_peg` function rather than adding `has_peg` to this one. for one example, we should add `peg_height = 0.05` as a keyword argument, but it doesn't feel right to add that as a kwarg to this existing pybullet block creator. and as a general rule of thumb, we can imagine other variations of blocks that people might want to create in the future, and we wouldn't want this create_pybullet_block function to handle all those variations -- it would get bloated and confusing pretty quickly.